### PR TITLE
chore: release-please initial files

### PR DESCRIPTION
**Pull request by terovirtanen**
release-please helps to create a release, main points:

- release PR
  - base on conventional commits
- increase version
- update changelog
- create github release
- create git tag

More details found from [Releases with release-please](https://helsinkisolutionoffice.atlassian.net/wiki/spaces/DD/pages/8278966368/Releases+with+release-please)

[OH-71](https://helsinkisolutionoffice.atlassian.net/browse/OH-71)

[OH-71]: https://helsinkisolutionoffice.atlassian.net/browse/OH-71?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .release-please-manifest.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  ".": "3.1.1"
```
Comment (harriris-vincit) [inline .release-please-manifest.json:2]:
```suggestion
  ".": "3.2.0"
```
I guess this is the current version of the project using release-please?

File: release-please-config.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "bootstrap-sha": "ac251ea3b034166f94172f932e10f5920b3aec42",
+  "sequential-calls": true,
+  "packages": {
+    ".": {
```
Comment (harriris-vincit) [inline release-please-config.json:5]: One question here would be that do we want to use the default `changelog-sections` option (see the default config at https://git.io/JqCZL) that hides, for example, `chore`, `refactor` and `docs` type of commits or would we rather want to show some or all of the hidden ones too?

I think `docs` would be good to be shown when the documentation has been updated (as we are also planning to bring the API Swagger documentation to be part of this repo). `chore` sometimes includes pretty noticable version updates like the Django upgrade.

File: .release-please-manifest.json (hunk 1 above)
Comment (terovirtanen) [reply .release-please-manifest.json]: version has set to 3.2.0

File: release-please-config.json (hunk 2 above)
Comment (terovirtanen) [reply release-please-config.json:5]: I created [OH-73](https://helsinkisolutionoffice.atlassian.net/browse/OH-73)
I think `changelog-sections` configuration should do in another PR

**Review by harriris-vincit — APPROVED** [review]
LGTM after discussions

**Pull request merged**